tests/Yaapii.Atoms.Tests/Scalar/OrTests.cs

```diff
@@ -44,6 +43,69 @@ public void TrueOrGenScalar()
                     ).Value() == true
             );
         }
-    }
-}
 
+        [Fact]
+        public void TestFunc()
+        {
+            Assert.True(
+                    new Or<int>(
+                        input => input > 0,
+                        1, -1, 0
+                    ).Value());
+        }
+
+        [Fact]
+        public void TestFunc2()
```

Comment:
Please make a name for the test which tells us what it tests

Reply:
Done